tests/UI/campaigns/functional/BO/03_catalog/06_files/01_CRUDFile.js

```diff
@@ -1,10 +1,14 @@
 require('module-alias/register');
 
+// Import expect from chai
```

Comment:
Never noticed that but, I feel like this comment is not useful at all, it's verbose enough to understand that you import expect from chai, wdyt @boubkerbribri ?

Reply:
It's not useful, could be deleted later :+1: